File: Java/Basics/file11.java

//given number is palindrome or not ..

import java.util.*;

public class file11 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int num = sc.nextInt();
        int rev = 0, org_num = num;

        while(num != 0) {
            int dgt = num % 10;
            rev = rev * 10 + dgt;
            num /= 10;
        }

        if(rev == org_num) {
            System.out.println("Palindrome");
        } else {
            System.out.println("Not Palindrome..");
        }
    }
}

File: C/Basics/file11.c

// given number is palindrome or not .

#include<stdio.h>

int main() 
{
    int num = 0, i, org_num = 0, rev = 0;

    scanf("%d", &num);
    org_num = num;

    while(num != 0) {
        int dgt = num % 10;
        rev = rev * 10 + dgt;
        num /= 10;
    }

    if(org_num == rev) {
        printf("Palindrome..\n");
    } else {
        printf("Not Palindrome...\n");
    }

    
}